.github/dockerhub-readme.yml

```
@@ -0,0 +1,32 @@
+name: dockerhub-readme
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+on:
+  push:
+    branches:
+      - 'main'
+    paths:
+      - '.github/dockerhub-readme.yml'
+      - 'docs/dockerhub.md'
+
+env:
+  DOCKERHUB_SLUG: distribution/distribution
+
+jobs:
+  update:
+    permissions:
+      contents: read
+
+    runs-on: ubuntu-latest
```

nit: can be moved right after `env` to affect the whole workflow

```suggestion
env:
  DOCKERHUB_SLUG: distribution/distribution

permissions:
  contents: read

jobs:
  update:
    runs-on: ubuntu-latest
```